src/Models/Services/Utils.ts

```diff
@@ -800,3 +802,21 @@ export const isValidADXClusterUrl = (clusterUrl: string): boolean => {
     }
     return false;
 };
+
+/** Creates mock time series data array with data points between now and a certain milliseconds ago */
+export const getMockTimeSeriesDataArrayInLocalTime = (
```

Comment:
since this is not really a generic util that the entire app needs to know about, can we put it in a local util file for your components instead of the general file?

Reply:
wait I just noticed that I am using it in another file, so will keep it in the utils